test-case/check-sof-logger.sh

```diff
@@ -48,23 +48,29 @@ fi
 dlogi "Found file: $(md5sum $ldcFile|awk '{print $2, $1;}')"
 
 tmp_file=/tmp/$RANDOM.sof-logger.log
+err_tmp_file=/tmp/err_logger.log
 
 func_lib_check_sudo
 
 dlogi "Try to dump the dma trace log via sof-logger ..."
-dlogc "sudo $loggerBin -t -l $ldcFile -o $tmp_file 2>&1 &"
-sudo $loggerBin -t -l $ldcFile -o $tmp_file 2>&1 &
+# sof-logger errors will output to $err_tmp_file
+dlogc "sudo $loggerBin -t -l $ldcFile -o $tmp_file 2> $err_tmp_file &"
+sudo $loggerBin -t -l $ldcFile -o $tmp_file 2> $err_tmp_file &
 sleep 2
 dlogc "sudo pkill -9 $(basename $loggerBin)"
 sudo pkill -9 $(basename $loggerBin) 2> /dev/null
 
-# get size of trace log$
```

Comment:
can we still keep the trace size? There will also be issue that FW do not send trace to kernel as we have tried to checked in previous. We will need the Error check before this.